scripts/import_map/update.ts

```diff
@@ -0,0 +1,18 @@
+import { parse } from "https://deno.land/std/flags/mod.ts";
+
+const flags = parse(Deno.args, {
+  string: ["import-map", "api"],
+  default: {
+    "import-map": `${Deno.cwd()}/import_map.json`,
+    "api": "../deno-slack-api/src/",
+  },
+});
+
+const importMap = JSON.parse(await Deno.readTextFile(flags["import-map"]));
+importMap["imports"]["deno-slack-api/"] = flags.api;
+importMap["scopes"] = {
+  "https://deno.land/x/deno-slack-sdk/": {
+    "deno-slack-api/": flags.api
```

Comment:
Did we verify this works? I entered a [similar issue on the Deno repo](https://github.com/denoland/deno/issues/18195)

Reply:
I've update how the script works and I think its is in a good place now 💯